### DISCUSSION TITLE
Import file not recognised by the app

### DISCUSSION THREAD

**Discussion by girishkdesh** [iSoron/uhabits#1876] [Help & FAQ]
I had my backup exported three times from the app. For a reason I had to uninstall the app. When I reinstalled it and tried to import the backup files, none worked. A notification pops up... The file is not recognised.
I tried unzipping the backup file and tried to feed the CSV files underneath, but doesn't work anyways.
I have a very long streak of precious data in the backup.
Please help restoring the data.
Here is the link to my backup file. Please find out what's wrong with that

https://drive.google.com/file/d/18PRj469TB1Cw4xss0cXeh8vCC3B47Lbh/view?usp=drivesdk

**Comment by ivanchoff** [comment]
Same problem here :(
it is a bug? it is  a normal behavior? why exported data can not be imported?

**Comment by psionicm** [comment]
I also have this issue

**Comment by KyleSCraig** [comment]
If your backup was the full backup which generates a .db file then you should be able to import it. But if you have a .zip with .CSV inside then that can't be imported. It says this in the app. "This file cannot be imported back" at the end of the text on the export button.

It's pretty clear OP has the CSV case. @psionicm can you confirm for yourself.

**Reply by psionicm** [reply]
Ah, a true case of the user (me) not having RTM... I withdraw my question on account of not having correctly read the instructions. Thank you @KyleSCraig .

**Reply by ZornHadNoChoice** [reply]
Not all hope is lost! The .csv files contain everything needed, except for notes I think. So it should be possible to convert them to the .db file, or at the very least, add the data manually.

**Reply by psionicm** [reply]
I did indeed add all the data manually.
